docs(cli/deploy): add the out of gas in deployment issue 🚗 (#2921)
latticexyz/mud: docs/pages/cli/deploy.mdx

```sh copy
export DEBUG=mud:cli:deploy
```

## Known issues

### Out of gas

When deploying to a blockchain, you might get an error due to the post-deploy script being out of gas:

```
Error: Error:
Transaction Failure: <transaction hash>
Error running "forge script PostDeploy --broadcast --sig run(address) <world address> --rpc-url <conduit URL> -vvv --legacy --no-gas-limit"
    at o (file:/// < directory > /tutorial3/node_modules/.pnpm/@latticexyz+common@2.0.12_@aws-sdk+client-kms@3.598.0_asn1.js@5.4.1_typescript@5.4.2_zod@3.23.8/node_modules/@latticexyz/common/dist/foundry.js:2:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    (...)
```

> For some reason, the default gas estimates coming from the blockchain are too low, so the transaction runs out of gas.

Reviewer: ```suggestion
In some cases, the `forge`'s default gas estimates are too low, so the transaction runs out of gas.
```

> To get around this issue, use the `--forgeScriptOptions` command line parameter to specify [the gas estimate multiplier](https://book.getfoundry.sh/reference/forge/forge-script?highlight=gas-estimate-multiplier#options) `forge script` will use.

Reviewer: ```suggestion
Until [the issue is fixed upstream](https://github.com/latticexyz/mud/issues/2907), you can get around it by using the `--forgeScriptOptions` command line parameter to specify [the gas estimate multiplier](https://book.getfoundry.sh/reference/forge/forge-script?highlight=gas-estimate-multiplier#options) `forge script` will use.
```
